File: software/examples/php/ExampleSimple.php
<?php

require_once('Tinkerforge/IPConnection.php');
require_once('Tinkerforge/BrickletMultiTouch.php');

use Tinkerforge\IPConnection;
use Tinkerforge\BrickletMultiTouch;

$host = 'localhost';
$port = 4223;
$uid = 'XYZ'; // Change to your UID

$ipcon = new IPConnection(); // Create IP connection
$mt = new BrickletMultiTouch($uid, $ipcon); // Create device object

$ipcon->connect($host, $port); // Connect to brickd
// Don't use device before ipcon is connected

// Get current touch state
$touchState = $mt->getTouchState();

if($touchState & (1 << 12)) {
	echo "In proximity, ";
}

if(($touchState & 0xfff) == 0) {
	echo "No electrodes touched\n\n";
} else {
	echo "Electrodes ";
	for($i = 0; $i < 12; $i++) {
		if($touchState & (1 << $i)) {
			echo $i . " ";
		}
	}
	echo "touched\n\n";
}

echo "Press key to exit\n";
fgetc(fopen('php://stdin', 'r'));
$ipcon->disconnect();

?>

File: software/examples/perl/example_simple.pl
#!/usr/bin/perl  

use Tinkerforge::IPConnection;
use Tinkerforge::BrickletMultiTouch;

use constant HOST => 'localhost';
use constant PORT => 4223;
use constant UID => 'jSN'; # Change to your UID

my $ipcon = Tinkerforge::IPConnection->new(); # Create IP connection
my $mt = Tinkerforge::BrickletMultiTouch->new(&UID, $ipcon); # Create device object

$ipcon->connect(&HOST, &PORT); # Connect to brickd
# Don't use device before ipcon is connected

# Get current touch state
my $touch_state = $mt->get_touch_state();
my $s = '';
my $left_shift_1_12_bits = (1 << 12);

if (($touch_state) & ($left_shift_1_12_bits))
{
    $s .= 'In proximity, ';
}

if (($touch_state & 0xFFF) == 0)
{
    $s .= 'No electrodes touched';
} 
else
{
    $s .= 'Electrodes ';
        
    for (my $i = 0; $i < 12; $i++)
    {
        my $left_shift_1_i_bits = 1 << $i;

        if ($touch_state & $left_shift_1_i_bits)
        {
            $s .= "$i ";
        }
    }

    $s .= 'touched';
}

print "$s\n";

print "Press any key to exit...\n";
<STDIN>;
$ipcon->disconnect();

